lib/tool_belt/commands/check_deprecation_warnings.rb

```diff
@@ -10,7 +10,9 @@ def execute
         config = ToolBelt::Config.new(config_file)
         options = config.options
         repo_path = "repos/#{options.namespace}/#{options.project}/"
-        raise "Repo path #{repo_path} not found!" unless File.exist?(repo_path)
+        unless File.exist?(repo_path)
+          raise "Repo path #{repo_path} not found! Did you forget to run './tools.rb setup-environment #{repo_path}' ?"
```

Comment:
`setup-environment` takes a config file, not a repo path. You could specify `config_file` here or keep it generic saying something like "for this release"

Reply:
@johnpmitsch Thanks, updated!